src/d3a/d3a_core/export.py

```diff
@@ -103,9 +133,24 @@ def _export_area_with_children(self, area: Area, directory: dir):
             subdirectory.mkdir(exist_ok=True, parents=True)
             for child in area.children:
                 self._export_area_with_children(child, subdirectory)
+                if isinstance(child.strategy, (CommercialStrategy, FinitePowerPlant)):
```

Comment:
This whole block should be moved to its own module that would calculate independently the KPI values